HIP/hip-21.md

```diff
@@ -32,41 +33,70 @@ Today, applications have to use the File Service and submit two file contents qu
 
 ## Rationale 
 
-The proposal is to create a free non-file HAPI query to return network information. As we continue to grow the network and transition into a permissionless model, the number of times the local address book will be required to update will increase. The transaction fee associated with the current file query hinders the ability to update the address book as frequently as it might be needed for applications to ensure their address book is always up to date. This information should be free. 
+The proposal is to create a rest API query to return network information. As we continue to grow the network and transition into a permissionless model, the number of times the local address book will be required to update will increase. The transaction fee associated with the current file query hinders the ability to update the address book as frequently as it might be needed for applications to ensure their address book is always up to date. This information should be free. 
 
 ## Specification 
 
-- Add a non-file HAPI query called **getNetworkInfo** to the existing Network Service
-- This query would not have a transaction fee
+REST API
+
+GET `api/v1/addressbooks`
```

Comment:
I think we should do `/api/v1/network/nodes` to model the thing(s) being returned and not the container of them. This also maps closer to the `NodeAddress` proto that would be the source of the information.